### PR TITLE
Fix frontend np.fmod

### LINKED ISSUE #9323: Fix arithmetic_operations.test_numpy_fmod
|   |   |
|---|---|
|paddle|<a href="https://github.com/unifyai/ivy/actions/runs/6408093971"></a>
|numpy|<a href="https://github.com/unifyai/ivy/actions/runs/6408417944"></a>
|jax|<a href="https://github.com/unifyai/ivy/actions/runs/6408417944"></a>
|tensorflow|<a href="https://github.com/unifyai/ivy/actions/runs/6408417944"></a>
|torch|<a href="https://github.com/unifyai/ivy/actions/runs/6408093971"></a>

### PR DESCRIPTION
**Pull request by hello-fri-end**
closes #9323

### CODE REVIEW

**Comment by hello-fri-end** [comment]
@zaeemansari70 Hi, can you review this, please?

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_arithmetic_operations.py [hunk 1]
```diff
@@ -649,29 +650,35 @@ def test_numpy_fmod(
...
-        x1=xs[0],
-        x2=xs[1],
-        out=None,
-        where=where,
-        casting=casting,
-        order="K",
-        dtype=dtype,
-        subok=True,
-    )
+    try:
```
Comment (zaeemansari70) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_arithmetic_operations.py:653]: These try except blocks can not be used here to catch the exceptions, you have to properly come up with a logic through hypothesis if it's a problem with the data generation, or you can fix the backend if the error is being raised by the backend.
The goal here is not to just make the test pass, but to fix the root cause of the issue which is making the test to fail.

**Review by zaeemansari70 — CHANGES_REQUESTED** [review]
Hey! First of all sorry for the delayed review, was caught up with some high priority tasks 😅  
Thank you soo much for contributing to ivy! 🙂
Your PR looks amazing ✨!
Just requested a few changes and left comments on them which needs to be addressed before we can merge your PR! 🙂
Thanks!

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_arithmetic_operations.py (hunk 1 above)
Comment (hello-fri-end) [reply ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_arithmetic_operations.py]: Hi @zaeemansari70! First of all, thanks for the review :rocket: 
I had seen these try and except blocks in other tests and thought maybe it was an acceptable solution. Actually, the problem here is that the divisor sometimes becomes 0 after casting to `dtype`. Now, I have added a condition that will ensure that such examples are ignored by hypothesis. Hope it's an acceptable solution :pray:

**Pull request merged**